### DISCUSSION TITLE
Closing the modal after a certain time

### DISCUSSION THREAD

**Discussion by erfan-goodarzi** [mantinedev/mantine#7206] [Feature requests]
It would be great if we have a prop something like `closeAfter`, Such as `closeOnClickOutside` and `closeOnEscape`.
We can set a time for a modal to close automatically.

**Comment by rtivital** [comment]
You can do that with setTimeout. It is outside the library scope.

**Comment by erfan-goodarzi** [comment]
Yup i know that we can do it by setTimeout but it could be helpful. So what is considered in its scope?
